virt/virt-4-8-release-notes.adoc

```diff
@@ -77,15 +75,17 @@ The SVVP Certification applies to:
 * Cloning a data volume into a different namespace is now faster and more efficient when using storage that supports Container Storage Interface (CSI) snapshots. The Containerized Data Importer (CDI) uses CSI snapshots, when they are available, to improve performance when you create a virtual machine from a template.
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
+* When a virtual machine frees space on a virtual disk, the discard requests are passed to the underlying storage device, possibly freeing up real storage capacity.
```

Comment:
When a virtual machine frees space on a virtual disk, by means of the fstrim or blkdiscard commands, the discard requests are passed to the underlying storage device, which may free up real storage capacity, provided the storage provider supports this feature.

Reply:
I've made the update based on the comment.